src/RockBridge.cpp

```diff
@@ -62,6 +69,11 @@ void RockBridge::Load(int _argc , char** _argv)
     RTT::corba::TaskContextServer::ThreadOrb(ORO_SCHED_OTHER, RTT::os::LowestPriority, 0);
 
     // Import typekits to allow RTT convert the types used by the components
+    RTT::types::TypekitRepository::Import(new orogen_typekits::rock_gazeboTypekitPlugin);
```

Comment:
I did this to match the headers include order... there is no special reason. I''ll call std first then.